articles/Setup-Linux-Python.md

```diff
@@ -48,13 +53,34 @@ On Ubuntu 16.04 install it like this:
 sudo apt-get install openmpi-bin
 ```
 
-Make sure that it's libraries can be found, e.g., by setting up `LD_LIBRARY_PATH`.
+Make sure that its libraries can be found, e.g., by setting up `LD_LIBRARY_PATH`.
 
 ## Installing CNTK for Python on Linux (Ubuntu 14.04 or Ubuntu 16.04)
 
-This page will walk you through the process of installing the Microsoft Cognitive Toolkit (CNTK) to use from Python in Linux (Ubuntu 14.04 or Ubuntu 16.04).
+This page will walk you through the process of installing the Microsoft Cognitive Toolkit (CNTK) to use from Python in Linux (Ubuntu 14.04 or Ubuntu 16.04). Note that Ubuntu 14.04 is supported for CNTK 2.3.1 and lower. All releases 2.4+ officially only support Ubuntu 16.04.
```

Comment:
The latest version of CNTK only supports ubuntu 16.04. ubuntu 14.04 is for older versions